.github/workflows/benchmark.yml

```diff
@@ -49,6 +49,8 @@ jobs:
         python -m pip install maturin
         rustup target add x86_64-unknown-linux-musl
         python -m pip install pytest
+        python -m pip install pytest-asyncio
+        python -m pip install trio
```

Comment:
what's trio for?

Reply:
running the tests locally wouldn't work without pytest-asyncio and trio. I'm not exactly sure why, if you can run them without these then I'll delete

Reply:
nvm fixed